helix-view/src/gutter.rs

```diff
@@ -108,6 +108,7 @@ pub fn diff<'doc>(
                 // we need to special case removals here
                 // these technically do not have a range of lines to highlight (`hunk.after.start == hunk.after.end`).
                 // However we still want to display these hunks correctly we must not yet skip to the next hunk here
+
```

Comment:
The extra space here seems unnessecary for this pr and adds an extra file to the changes that doesn't need to be changed